self-updating-cache/action.yml

```diff
@@ -0,0 +1,65 @@
+name: "Self Updating Cache"
+description: "Keeps and updated cache leveraging actions/cache and the recipe for using updated caches (https://github.com/actions/cache/blob/main/tips-and-workarounds.md#update-a-cache)"
```

Comment:
Nit: I think you meant either "Keeps an updated" or "Keeps and updates"

Reply:
Thanks! I did mean "Keeps an updated"